src/bun.js/api/bun/process.zig

```diff
@@ -970,6 +970,11 @@ pub const PosixSpawnOptions = struct {
     argv0: ?[*:0]const u8 = null,
     stream: bool = true,
 
+    /// Linux-only.
+    ///
+    /// Send a signal to the parent process when the child process exits.
+    deathsig: bun.SignalCode = @enumFromInt(0),
```

Comment:
bad practice doing hiding an invalid state in a valid enum. should zero be added to the enum as `none`?